File: two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        m = {}
        for i in range(0,len(nums)):
            d = target-nums[i]
            if d in m:
                return {i,m[d]}
            else:
                m[nums[i]]=i
        

File: two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map <int,int> m;
        for(int i=0;i<nums.size();i++)
        {
            int d=target-nums[i];
            if(m.count(d))
                return {i,m[d]};
            else
                m.insert(make_pair(nums[i],i));
        }
        return {0,0};
    }
};
